frontend/src/components/Pages/InspectionPage/InspectionSection.tsx

```diff
@@ -60,26 +117,84 @@ export interface DeckMissionType {
     }
 }
 
-export interface OngoingMissionType {
+export interface DeckMissionCount {
+    [color: string]: {
+        count: number
+        message: string
+    }
+}
+
+export interface ScheduledMissionType {
     [missionId: string]: boolean
 }
 
+interface IInspectionProps {
+    refreshInterval: number
+    updateScheduledMissionsMap: (areaMissions: DeckMissionType) => Promise<void>
+    scheduledMissions: ScheduledMissionType
+    ongoingMissions: ScheduledMissionType
+}
+
+const getDeadlineInspection = (deadline: Date) => {
+    const deadlineDays = getDeadlineInDays(deadline)
+    switch (true) {
+        case deadlineDays <= 1:
+            return 'red'
+        case deadlineDays > 1 && deadlineDays <= 7:
+            return 'red'
```

Comment:
```suggestion
        case deadlineDays <= 7:
            return 'red'
```

Reply:
I don't what's happening here but this code doesn't look like this anymore

Reply:
![image](https://github.com/equinor/flotilla/assets/108514046/008fc339-59ec-4f00-9d32-9283f30760de)

Reply:
It seems like the code has been updated now